README.md

```diff
@@ -69,8 +69,10 @@ The initialization of the database, including the setup of the `pgvector` extens
 
 ### Configuration
 
-- `cp .env.example .env`
+- `cp .env.example .env` or rename `.env.example` into .env
```

Comment:
Might be best to stick with the `cp` for documentation purposes in case users want to fork/continue using this git repo. (`.env` shouldn't be committed to source control with creds/secrets so typically you would leave the `.env.example` in the repo).

Reply:
.env is listed in gitingore so it wouldnt be pushed. I will remove the line as it is understandable the renaming would work too